REST_API_DOCS.md

```
@@ -0,0 +1,1861 @@
+**Request Headers**
+
+| 헤더           | 타입     | 필수 | 설명                                    |
+|--------------|--------|----|---------------------------------------|
+| Content-Type | string | O  | 요청 데이터를 `application/json` 형태로 전송합니다. |
+
+**Request Body**
+
+| 필드                                                                                                                   | 타입     | 필수 | 설명                                                                                                                                                                                                                      |
+|----------------------------------------------------------------------------------------------------------------------|--------|----|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| email                                                                                                                | string | O  | 회원가입 시 필요한 이메일 입니다. 1자 이상 255자 이하의 문자열이어야 하며, 유효한 이메일 형식을 입력해야합니다. 같은 이메일을 가진 리소스는 생성 불가능합니다.                                                                                                                           |
+| password                                                                                                             | string | O  | 회원가입 시 필요한 비밀번호입니다. 비밀번호 규칙을 준수해야합니다.                                                         <br> - 비밀번호 규칙: 8~16자의 영문 대소문자, 숫자 및 특수문자 사용 (사용 가능한 특수문자 32자 : ! " # $ % & ' ( ) * + , - . / : ; < = > ? @ [ ₩ ] ^ _ ` { | } ~)   |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 4; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)